pallets/automation-time/src/lib.rs

```diff
@@ -316,23 +313,12 @@ pub mod pallet {
 			task_id: TaskIdV2,
 			amount: BalanceOf<T>,
 		},
-		AutoCompoundDelegatorStakeFailed {
-			task_id: TaskIdV2,
-			error_message: Vec<u8>,
-			error: DispatchErrorWithPostInfo,
-		},
 		/// The task could not be run at the scheduled time.
 		TaskMissed {
 			who: AccountOf<T>,
 			task_id: TaskIdV2,
 			execution_time: UnixTime,
 		},
-		/// The result of the DynamicDispatch action.
-		DynamicDispatchResult {
-			who: AccountOf<T>,
-			task_id: TaskIdV2,
-			result: DispatchResult,
-		},
 		/// The call for the DynamicDispatch action can no longer be decoded.
```

Comment:
We don’t need this CallCannotBeDecoded event.

When CallCannotBeDecoded error happens, the below two Events need to capture it
1. TaskExecutionFailed
2. TaskNotRescheduled.

Reply:
Fixed